.github/workflows/automerge.yml

```diff
@@ -26,6 +26,28 @@ jobs:
             GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
             PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
 
+        # This Slack step is duplicated in all workflows, if you make a change to this step, make sure to update all
```

Comment:
Should we remove all the other instances? Can we exit the job here as well?

Reply:
Rearranged so we fail the workflow _then_ this announce step happens. However, I think we should leave the other announce steps in case the 3rd-party auto-approve and auto-merge actions fail for some other reason we're not aware of.